src/scss/layout/_anchor-link.scss

```
@@ -0,0 +1,59 @@
+@use '~accoutrement/sass/tools';
+
+.anchor-link-wrapper {
+  display: inline-block;
+  position: relative;
+
+  h2 {
+    display: inline-block;
+    padding-left: var(--anchor-link-padding-l, var(--gutter));
+
+    @include tools.above('wide-page') {
+      --anchor-link-padding-l: 0;
+    }
+  }
+}
+
+.header-anchor {
+  position: absolute;
+  left: var(--anchor-position-offset-l, -0.25em);
+  top: var(--anchor-position-offset-t, var(--gutter-plus));
+  transform-origin: top center;
+  width: var(--gutter);
+
+  @include tools.above('wide-page') {
+    --anchor-position-offset-l: -1em;
+    --anchor-position-offset-t: 1.25em;
+  }
+
+  &.anchor-clicked {
+    @include tools.transition('move');
+
+    svg {
+      transform: rotate(-45deg);
+    }
+  }
```

@dvdherron 

```suggestion
  svg {
    @include tools.transition('move');
  }
```

and then I'd move the svg transform to the .anchor-clicked (see comment a few steps belore that refers to the animation.)

```
.anchor-clicked {
...

  svg {
    transform: rotate(-45deg);
  }
...

```

---

fixed